src/deepsparse/tasks.py

```diff
@@ -166,6 +166,14 @@ def is_nlp(cls, task: str) -> bool:
         """
         return any([nlp_task.matches(task) for nlp_task in cls.nlp])
 
+    @classmethod
+    def is_cv(cls, task: str) -> bool:
+        return (
+            cls.is_yolo(task)
+            or cls.is_yolact(task)
+            or cls.is_image_classification(task)
```

Comment:
Python can be so beautiful

Reply:
there's nothing better than writing some beautiful code 😁